README.md

```diff
@@ -45,3 +45,11 @@ They are:
 	- Example: `[{"address":"1e2345...",rank: 4, "githubHandle":"user-1"},{"address":"1e4532...",rank: 3, "githubHandle":"user-2"},{"address":"1e8335...",rank: 6}]`
 - `github-handles`: All the fellows handles separated by commas
 	- Example: `user-1,user-2,user-3`
+
+## Adding github account to your on-chain identity
+
+Adding the github account to your identity requires the usage of the `additional` fields in the identity info. This is currently only supported by using the [polkadot-js bare extrinsic interface](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.polkadot.io#/extrinsics). See the following image for an example:
+
+![Add github name to the additional fields of the on-chain identity](.github/github-on-chain-identity-process.jpg)
+
+You should also at least add the `display` field. If you want to add `utf8` characters, you need to hex encode them and put the hex string (`0x` prefixed) into the field.
```

Comment:
It's probably worth mentioning that if the user has some of those fields already configured on-chain, then sending a transaction with `None` will override those fields to an empty value.

So you cannot just add the github username, you have to include all other already-existing field values as well.